### DISCUSSION TITLE
Using with a HttpOnly cookie from server

### DISCUSSION THREAD

**Discussion by chrisbekas** [react-auth-kit/react-auth-kit#1022] [General] [answered]
Wondering if authkit can be used by having the JWT in HttpOnly cookie sent by the server?

**Comment by darkmatter18** [comment, marked as answer]
Yes, you can use Auth kit by sending the cookie in the initial response from the server. But you have to make sure 2 things

1. The cookies sent by the server should have the same configuration settings as like Auth Kit.
2. The cookie must be set before any JavaScript start running.

Hopefully, this will answer your issue. Thanks!

**Reply by chrisbekas** [reply]
Thanks for that!
